reels/fluter/__init__.py

```diff
@@ -19,15 +19,17 @@
 
 
 class Fluter:
-    def __init__(self, host=None, width=16, height=24, depth=3):
+    DEFAULT_HOST = None
+
+    def __init__(self, host=DEFAULT_HOST, width=16, height=24, depth=3):
         self.host, self.port = self._parse_host(host)
         self.width = width
         self.height = height
         self.depth = depth
         self.socket = None
 
     def _parse_host(self, host):
-        if not host:
+        if host is Fluter.DEFAULT_HOST:
             host = os.environ.get("PIXELFLUT_HOST", "localhost:1234")
```

Comment:
You can use `Fluter.DEFAULT_HOST` as default value for `.get` when you define `DEFAULT_HOST= "localhost:1234"` at the top of this class.

Reply:
same reasoning as above